README.md

```diff
@@ -59,16 +59,28 @@ curl -s https://packagecloud.io/install/repositories/lukaspustina/opensource/scr
 sudo apt-get install wpscan-analyze
 ```
 
-### Install script
+### Install script [All Unix platforms]
 
 **Simply run:**  
 ```bash
-curl -s https://raw.githubusercontent.com/lukaspustina/wpscan-analyze/master/install.sh | sh
```

Comment:
I'd rather understand why the `curl | sh` does not work. The usual suspect is that you make assumption about what `sh` actually is. It might be bash, it might dash ... etc.

You could try to check that with your Raspi and run `shellcheck` to verify if you have any incompatibilities or ambiguous syntax -- see below.

Reply:
The install script use `sh` by purpose.
It also works with `bash` tho.
I'll fix the other problems and see if that's an issue

Reply:
I'm not able to make it work with `curl | bash`.  
Please feel free to add commit to https://github.com/tristanlatr/wpscan-analyze/tree/patch-1 if tou want to edit the script